kernel/trace/bpf_trace.c

```diff
@@ -326,6 +327,9 @@ static const struct bpf_func_proto bpf_probe_read_compat_str_proto = {
 BPF_CALL_3(bpf_probe_write_user, void __user *, unsafe_ptr, const void *, src,
 	   u32, size)
 {
+	printk("bpf_probe_write_user is disabled for security reasons");
```

Comment:
Will it work simply commenting out lockdowns from `lockdown_reason` array in the `security/security.c` file?